src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+
+        FileReaderImpl fileReader = new FileReaderImpl();
+        List<String> inputReport = fileReader.read("reportToRead.csv");
```

Comment:
Ensure that the input file path 'reportToRead.csv' is placed in the `src/main/resources` folder as per the checklist requirement: 'Place the input and output files into the `src/main/resources` folder.'